packages/algob/src/lib/account.ts

```diff
@@ -63,3 +63,12 @@ export function validateAccount (a: Account, errs: ErrorPutter): boolean {
   if (!(typeof a.name === 'string' && a.name !== "")) { errs.push("name", "can't be empty", 'string'); }
   return errs.isEmpty;
 }
+
+export function mkAccountIndex (accountList: Account[]): Accounts {
+  return accountList.reduce(
+    (out: Accounts, a: Account) => {
+      out[a.name] = a;
+      return out;
+    },
+    {});
```

Comment:
The type is wrong. It should be `Map<>`

Side note: using `for..of` loop would be shorter and easier for newcomers.

Reply:
Updated to Map.

Reply:
`map` should we use for homomoprhic operations between containers. Moreover it's not more readable. So, I would prefer to stay with `reduce` then with `map`.